src/framework/shortcuts/internal/platform/macos/macosshortcutsinstancemodel.mm

```diff
@@ -352,13 +304,10 @@ static QString translateToCurrentKeyboardLayout(const QKeySequence& sequence)
         return {};
     }
 
-    Qt::KeyboardModifiers modifiers = keyCombination.keyboardModifiers();
-    UInt32 keyNativeModifiers = nativeModifiers(modifiers);
-
     QString keyStr = keyCodeToString(keyboard, keyNativeCode);
-    QString modifStr = keyModifiersToString(keyNativeModifiers);
```

Comment:
Do we need these changes? Could you please explain?

Reply:
We need to return `QKeySequence` from this method to be able to get consistent modifier order. It seems to make no sense to convert from Qt modifiers to native modifiers and then from native modifiers back to Qt string notation, and then from Qt string notation to QKeySequence. Instead, we can go directly from Qt modifiers to QKeySequence.

Reply:
we need to think about how risky this is for the patch

Reply:
Owing to the capricious nature of shortcuts, I suspect this warrants more extensive testing than we currently have time for in 4.4.3. Let's target this for 4.5.